### DISCUSSION TITLE
How to use storage-resize-images extension for multiple storage buckets in one firebase project

### DISCUSSION THREAD

**Discussion by mussacharles60** [firebase/extensions#763] [Q&A]
I'm working on a project which I use multiple storage buckets, each has its own files including images... so I want to use the `storage-resize-images` extension for every bucket in my project. Please help on how to do that. thanks

**Comment by mtc-jed** [comment]
Bumping this, same problem.

**Reply by i14h** [reply]
Could you install multiple instances of the extension?

**Reply by mtc-jed** [reply]
I would have to do that automatically when a new bucket is created, don't know if that's possible. Also that would mean the day I need to change the config, I have to do it on all instances. It a temporary solution but a bad one, but I'd rather do it myself in a cloud function if it's the only way of solving this.

**Reply by mussacharles60** [reply]
I didn't know if you can install multiple instances, but if it works, that's great
